### ISSUE TITLE
It's time for 1.0.0 😁👍

### ISSUE THREAD

**Issue by jedwards1211** [apollographql/subscriptions-transport-ws#676]
This package is over 2 years old and it's even been 2 years since the minor version number was bumped.  So one can hardly say this package is still beta.  Until version 1.0.0 is released there's no capability to distinguish between bugfixes and new features in version numbers.

**Comment by gklijs** [comment]
I think with 160 open issues, there is some work to do before a 1.0.0...

**Comment by jedwards1211** [comment]
It really doesn't matter how many issues there are.  The version can be bumped as issues get fixed.

[How do I know when to release 1.0.0?](https://semver.org/#how-do-i-know-when-to-release-100)
If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backwards compatibility, you should probably already be 1.0.0.

**Comment by gklijs** [comment]
Might make sense, it's never bug free. I don't really care much for the first number being a zero. Although when it's not, in general iets more likely to have breaking changes. Only but always up the first number on a breaking change would by ideal I think.

**Comment by jedwards1211** [comment]
semver considers a middle number increase in 0.x versions a breaking change, so breaking changes actually tend to be more rampant in packages on 0.x versions.

**Comment by razor-x** [comment]
Check out how the `graphql` package addressed this issue: https://medium.com/@leeb/graphql-js-preparing-for-v14-0-0-839f823c144e. You can see the discussion leading up to it here: https://github.com/graphql/graphql-js/issues/1005.

TLDR, we see the same process play out over so many widely adopted packages used in production, and users really appreciate it when maintainers move out from the pre-release version. Users and maintainer shouldn't have some magical bar for 1.0.0. That's why it's part of the spec (to take the stress off of everyone involved) and it's oh-so-simple:

>  If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backwards compatibility, you should probably already be 1.0.0.

We just want the version number to be a useful tool. When a "semver" project sits at 0.x for too long after it's already earned a 1.0, it just [becomes a meme](https://0ver.org/).

**Comment by glasser** [comment]
This project is not actively maintained currently.

The `graphql-ws` package (originally a fork of this package) may be a better alternative for many users
as it is actively maintained. It uses a different wire protocol, so you will need to ensure that all integrations (client, server, IDE, etc) use it.

**Comment by gklijs** [comment]
@glasser just to be sure, https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md you mean right?

**Comment by glasser** [comment]
Yes.

An unfortunately confusing fact is that the actual `protocol` name (as in `sec-websocket-protocol`) used by the graphql-ws package is graphql-transport-ws whereas the protocol used by subscriptions-transport-ws is graphql-ws...
